main.go

```diff
@@ -35,7 +33,7 @@ var (
 	findBlock            = app.Command("find-block", "Finds the block height for a given date/time.")
 	findBlockTimestamp   = findBlock.Arg("timestamp", "date/time to resolve. E.g. \"2006-01-02 15:04:05 MST\"").Required().String()
 	findBlockBackend     = findBlock.Flag("backend", "electrum | btcd | electrum-recorder | btcd-recorder | fixture").Default("electrum").Enum("electrum", "btcd", "electrum-recorder", "btcd-recorder", "fixture")
-	findBlockAddr        = findBlock.Flag("addr", "Backend to connect to initially. Defaults to a hardcoded node for Electrum and localhost for Btcd.").PlaceHolder("HOST:PORT").TCP()
+	findBlockAddr        = findBlock.Flag("addr", "Backend to connect to initially. Defaults to a hardcoded node for Electrum and localhost for Btcd.").PlaceHolder("HOST:PORT").String()
```

Comment:
why this change? do we want to use electrum `{s,t}<PORT_NUM>` here?

Reply:
We don't really have a choice, unless we add another flag.